superbench/benchmarks/micro_benchmarks/cublas_function.py

```diff
@@ -291,7 +291,7 @@ def _process_raw_result(self, cmd_idx, raw_output):
                     raw_data = raw_data.split(',')
                     raw_data.pop()
                     raw_data = [float(item) for item in raw_data]
-                    self._result.add_result(metric, statistics.mean(raw_data))
```

Comment:
Unify the `statistics.mean()` and `sum(raw_data) / len(raw_data)`?

Reply:
unify to statistics.mean()